scipy/interpolate/interpolate.py

```diff
@@ -661,8 +661,13 @@ def __init__(self, c, x, extrapolate=None, axis=0):
             extrapolate = bool(extrapolate)
         self.extrapolate = extrapolate
 
+        if self.c.ndim < 2:
+            mesg = "Coefficients array must be at least 2-dimensional."
```

Comment:
`mesg` looks a bit odd, I believe you meant `msg`?

By the way, why to use another variable and not just pass the string into `ValueError`?